.travis/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+cd $TRAVIS_BUILD_DIR
```

Comment:
Just curious, why do we need this?

Reply:
To not adjust all paths in commands, eg. `pytest -v ../tests/ --cov=../m2cgen/ --ignore=../tests/e2e/`.